backend/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "aquatica",
+  "version": "1.0.0",
+  "description": "Welcome to **Aquatica** – an e-commerce platform designed specifically for aquarium enthusiasts! Whether you’re a beginner looking to set up your first tank or an experienced aquarist seeking rare fish species, Aquatica is the go-to marketplace for all your aquarium needs.",
+  "main": "server.js",
+  "scripts": {
+    "start": "node server.js",
```

Comment:
When I tried to run npm start, I got this error message:

`(node:14692) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
(Use `node --trace-warnings ...` to show where the warning was created)
/Users/winifrednauda/Class/COMPSCI326/Aquatica/backend/server.js:2
import express from 'express';
^^^^^^

SyntaxError: Cannot use import statement outside a module
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1149:20)
    at Module._compile (node:internal/modules/cjs/loader:1190:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1280:10)
    at Module.load (node:internal/modules/cjs/loader:1089:32)
    at Module._load (node:internal/modules/cjs/loader:930:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:81:12)
    at node:internal/main/run_main_module:23:47`